common-rendering/storybook-static/project.json

```diff
@@ -0,0 +1 @@
+{"generatedAt":1669218882584,"builder":{"name":"webpack5"},"hasCustomBabel":false,"hasCustomWebpack":true,"hasStaticDirs":false,"hasStorybookEslint":false,"refCount":0,"monorepo":"Workspaces","packageManager":{"type":"yarn","version":"1.22.19"},"features":{"buildStoriesJson":true},"storybookVersion":"6.4.9","language":"typescript","storybookPackages":{"@storybook/addons":{"version":"6.5.13"},"@storybook/api":{"version":"6.5.13"},"@storybook/builder-webpack5":{"version":"6.4.9"},"@storybook/components":{"version":"6.5.13"},"@storybook/core-events":{"version":"6.5.13"},"@storybook/manager-webpack5":{"version":"6.4.9"},"@storybook/react":{"version":"6.4.9"},"@storybook/theming":{"version":"6.5.13"}},"framework":{"name":"react"},"addons":{"@storybook/addon-essentials":{"version":"6.4.9"},"@storybook/addon-knobs":{"version":"6.4.0"}}}
```

Comment:
not sure why these aren't gitignored like the rest of the `static-storybook` folder